src/main/kotlin/com/mineinabyss/launchy/ui/ModInfo.kt

```diff
@@ -87,6 +84,30 @@ fun ModInfo(group: Group, mod: Mod) {
                             contentDescription = "URL"
                         )
                     }
+                if (mod.configUrl != null) {
+                    var isEnabled by remember { mutableStateOf(configEnabled) }
```

Comment:
Like you mention, this needs to be moved into the state class and persisted similar to the other properties.